Frends.AzureEventHub.Receive/Frends.AzureEventHub.Receive/Definitions/Consumer.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureEventHub.Receive.Definitions;
+
+/// <summary>
+/// Consumer parameters.
+/// </summary>
+public class Consumer
+{
+    /// <summary>
+    /// Authentication method.
+    /// </summary>
+    /// <example>AuthenticationMethod.ConnectionString</example>
+    [DefaultValue(AuthenticationMethod.ConnectionString)]
+    public AuthenticationMethod AuthenticationMethod { get; set; }
+
+    /// <summary>
+    /// The name of the specific Event Hub.
+    /// </summary>
+    /// <example>ExampleHub</example>
+    public string EventHubName { get; set; }
+
+    /// <summary>
+    /// The name of the consumer group. 
+    /// Events are read in the context of this group.
+    /// Using default consumer group if left empty.
+    /// </summary>
+    /// <example>$Default</example>
+    public string ConsumerGroup { get; set; }
+
+    /// <summary>
+    /// The connection string.
+    /// </summary>
+    /// <example>Endpoint=sb://NamespaceName.servicebus.windows.net/;SharedAccessKeyName=KeyName;SharedAccessKey=KeyValue</example>
+    [UIHint(nameof(AuthenticationMethod), "", AuthenticationMethod.ConnectionString)]
+    [PasswordPropertyText]
```

Comment:
Please add annotation for [DisplayFormat(DataFormatString = "Text")] otherwise the input field can default to Expression.